test/inductor/test_indexing.py

```diff
@@ -231,12 +226,10 @@ def test_print_ceil(self):
                 self.assertExpectedInline(cexpr(expr), """std::ceil((1.0/2.0)*s1)""")
 
     def test_print_round(self):
-        expr = Round(sympy.Symbol("x", integer=True) / 2)
+        expr = RoundToInt(sympy.Symbol("x", integer=True) / 2)
         self.assertExpectedInline(pexpr(expr), """round((1/2)*x)""")
         self.assertExpectedInline(cexpr(expr), """std::lrint((1.0/2.0)*x)""")
-        self.assertExpectedInline(
-            texpr(expr), """libdevice.llrint((1/2)*x).to(tl.int64)"""
```

Comment:
https://docs.nvidia.com/cuda/libdevice-users-guide/__nv_llrint.html#__nv_llrint

llrint already returns int64 so the conversion to int64 was unnecessary